tt_metal/hw/inc/wormhole/noc_nonblocking_api.h

```diff
@@ -170,13 +168,13 @@ inline __attribute__((always_inline)) void ncrisc_noc_fast_write_l1(uint32_t noc
   }
 }
 
-inline bool ncrisc_noc_fast_write_ok(uint32_t noc, uint32_t cmd_buf) {
-  return (NOC_CMD_BUF_READ_REG(noc, cmd_buf, NOC_CMD_CTRL) == NOC_CTRL_STATUS_READY);
-}
+// inline __attribute__((always_inline)) bool ncrisc_noc_fast_write_ok(uint32_t noc, uint32_t cmd_buf) {
+//   return (NOC_CMD_BUF_READ_REG(noc, cmd_buf, NOC_CMD_CTRL) == NOC_CTRL_STATUS_READY);
+// }
```

Comment:
Can this be removed instead of commented out?

Reply:
wanted to keep it for a bit at least so that its clear that these are the functions being replaced.